code/modules/mob/living/carbon/xenomorph/castes/defender/abilities_defender.dm

```diff
@@ -147,6 +147,44 @@
 		X.use_plasma(plasma_cost)
 	add_cooldown()
 
+// ***************************************
+// *********** Forward Charge
+// ***************************************
+/datum/action/xeno_action/activable/forward_charge
+	name = "Forward Charge"
+	action_icon_state = "charge"
+	mechanics_text = "Charge up to 4 tiles and knockdown any targets in your way."
+	ability_name = "charge"
+	cooldown_timer = DEFENDER_CHARGECOOLDOWN

```

Comment:
```suggestion
	cooldown_timer = 15 SECONDS
```

Reply:
Why when everything is done via a define?

Reply:
this is needless obfuscation

Reply:
nothing else is referencing the define.

Reply:
It's the standard we've been following, but we can change it. Spooky's suggestion is better.